nixos/modules/virtualisation/anbox.nix

```diff
@@ -5,7 +5,9 @@ with lib;
 let
 
   cfg = config.virtualisation.anbox;
-  kernelPackages = config.boot.kernelPackages;
+  inherit (config.boot) kernelPackages;
+  inherit (kernelPackages) kernel;
```

Comment:
Is that even used?

Reply:
No, it's a leftover from squashing a few patches.